### ISSUE TITLE
can't work in unity 2019

### ISSUE THREAD

**Issue by caochao** [mythosMatheWG/unityIntoIOSSample#1]
hi Markus, I follow your sample codes to add unity view in a ios native view with unity 2019.1,, it can't work cause can't get unity rootViewController, see what happens:
![image](https://user-images.githubusercontent.com/1911170/58680096-e0b90400-8398-11e9-9570-d0ba7976d466.png)

and finaly i achive this task in below methods:
![image](https://user-images.githubusercontent.com/1911170/58680189-3f7e7d80-8399-11e9-8787-1e6aede11109.png)

**Comment by kuegi** [comment]
yes. this sample was done in unity 5.1.3 . there were probably some changes along the way to 2019.1
feel free to add a pullrequest with the necessary changes for 2019.1
